.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "${workspaceRoot}/.venv/bin/python"
```

Comment:
I am not sure if it is desirable to force all VSCode users to have the same settings.